src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/MarshalMethodTests.cs

```diff
@@ -0,0 +1,199 @@
+#nullable enable
```

Comment:
https://github.com/dotnet/android/pull/10093 was merged should we rebase/remove this line?